constellations/gemini-user/src/internship/internship.test.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import { InternshipPage } from "./internship.js";
+
+describe("Internship Page", () => {
+    const component = shallow(<InternshipPage />);
+
+    test("renders", () => {
+        expect(component.exists()).toBe(true);
+        expect(component.find("h1").text()).toContain(
+            "Software Development Internship"
+        );
+        expect(component.find("h1").text()).toContain("Technology Stack");
```

Comment:
I think you can specifiy specific `<h1>` to contain certain text.

So the first h1 should have "Software Development Internship". The second should have Technology Stack and so on...

I think you can use `component.find("h1").at(0)` using indexes.